dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/controller/RequestController.kt

```diff
@@ -163,6 +163,7 @@ class RequestController(
         message: String?,
         requestPriority: RequestPriority?,
         adminComment: String?,
+        requestStatusChangeReason: String?,
```

Comment:
Not sure where in the code the error is but the problem is that uploading a data set does no longer result in the Request being moved from Open to Answered.